src/events/message.js

```diff
@@ -1,6 +1,11 @@
 const moment = require('moment');
 const { locale } = require('../../config/locale');
 const { prefix } = require('../../config/bot');
+const {
+  channels: {
```

Comment:
I'm not a huge of this nested destructure. It can lead to confusion.

Much prefer to use `channels.TRIGGERS_CHANNEL_ID`, or destructure down there where needed.